package.json

```diff
@@ -95,10 +93,8 @@
     "randombytes": "2.1.0"
   },
   "devDependencies": {
-    "bitcore-build": "bitpay/bitcore-build",
-    "brfs": "^1.2.0",
+    "mocha": "^5.2.0",
```

Comment:
That's the version that has already been present in the deptree and used for the `test` command above.